packages/forms/src/UIForm/fields/Button/Buttons.component.js

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import FieldTemplate from '../FieldTemplate';
+import SingleButton from './SingleButton.component';
+
+import theme from './Buttons.scss';
+
+export default function Buttons(props) {
+	const { id, onTrigger, schema } = props;
+	return (
+		<FieldTemplate description={schema.description}>
+			<div className={theme.buttons}>
+				{
+					schema.items &&
+					schema.items.map((itemSchema, index) => (
+						<SingleButton
+							className={theme[itemSchema.position]}
+							key={index}
+							id={`${id}-${index}`}
+							onTrigger={onTrigger}
+							schema={itemSchema}
+						/>
+					))
```

Comment:
Could you extract that part to a function since it contains both a conditional and a map?